code/modules/antagonists/heretic/magic/rust_charge.dm

```diff
@@ -0,0 +1,25 @@
+// Rust charge, added so i could put the lion hunter rifle earlier up the tree
+/datum/action/cooldown/mob_cooldown/charge/rust
+	name = "rust charge"
+	desc = "A charge that must be started on a rusted tile and will destroy any rusted objects you come into contact with, will deal high damage to others and rust around you during the charge."
+	var/charge_distance = 10
```

Comment:
```suggestion
	charge_distance = 10
```
You don't need to add `var/` since the parent (`/datum/action/cooldown/mob_cooldown/charge`) already declared it.